src/include/daos/tests_lib.h

```diff
@@ -206,4 +206,18 @@ dts_create_config(char buf[DTS_CFG_MAX], const char *format, ...)
 		buf[DTS_CFG_MAX - 1] = 0;
 }
 
+static inline void
+dts_append_config(char buf[DTS_CFG_MAX], const char *format, ...)
+{
+	va_list	ap;
+	int	count = strlen(buf);
```

Comment:
should use strnlen here

Reply:
Good catch.

In the case count == DTS_CFG_MAX already, vsnprintf(buf+count, 0, ...) is safe cause no write to the buf with size 0.